*** tests/A-11-01-M-ok.out	2021-11-07 16:04:59.127560000 +0000
--- tests/A-11-01-M-ok.outhyp	2021-11-09 23:57:10.641159538 +0000
***************
*** 69,75 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|1|M1|SAL|10|1000|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 69,75 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|1|M1|SAL|10|100|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
