*** tests/A-12-10-M-ok.out	2021-11-18 17:29:53.847835943 +0000
--- tests/A-12-10-M-ok.outhyp	2021-11-18 20:36:10.011034112 +0000
***************
*** 83,89 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -50
  Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
--- 83,89 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -500
  Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
***************
*** 121,127 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -150
  Saldo contabilístico: -100
  Menu Principal
  1 - Abrir
--- 121,127 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -600
  Saldo contabilístico: -100
  Menu Principal
  1 - Abrir
