*** tests/A-12-08-M-ok.out	2021-11-18 17:29:53.846835941 +0000
--- tests/A-12-08-M-ok.outhyp	2021-11-18 20:36:09.397034711 +0000
***************
*** 58,64 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|4|M1|GARRAFA|10|100|100|0|VIDRO:10:30#ROLHA:10:20
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 58,64 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|4|M1|GARRAFA|10|90|90|0|VIDRO:10:30#ROLHA:10:30
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 77,84 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 120
! Saldo contabilístico: 600
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 77,84 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 142
! Saldo contabilístico: 642
  Menu Principal
  1 - Abrir
  2 - Guardar
