*** tests/A-12-07-M-ok.out	2021-11-19 18:36:45.613724921 +0000
--- tests/A-12-07-M-ok.outhyp	2021-11-19 18:37:31.168492461 +0000
***************
*** 81,87 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 81,87 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 230
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 118,125 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 53
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 118,125 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 52
! Saldo contabilístico: 230
  Menu Principal
  1 - Abrir
  2 - Guardar
