*** tests/A-12-09-M-ok.out	2021-11-18 17:29:53.847835943 +0000
--- tests/A-12-09-M-ok.outhyp	2021-11-18 20:36:09.788034329 +0000
***************
*** 58,64 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|4|M1|GARRAFA|10|-20|0|0|VIDRO:10:30#ROLHA:10:20
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 58,64 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|4|M1|GARRAFA|10|-30|0|0|VIDRO:10:30#ROLHA:10:30
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 77,84 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 140
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 77,84 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 22
! Saldo contabilístico: 162
  Menu Principal
  1 - Abrir
  2 - Guardar
