*** tests/A-12-06-M-ok.out	2021-11-18 17:29:53.846835941 +0000
--- tests/A-12-06-M-ok.outhyp	2021-11-18 20:36:09.013035085 +0000
***************
*** 69,75 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 130
  Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
--- 69,75 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
  Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
***************
*** 107,113 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 180
  Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
--- 107,113 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
  Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
