*** tests/A-14-04-M-ok.out	2021-11-18 17:29:53.847835943 +0000
--- tests/A-14-04-M-ok.outhyp	2021-11-18 20:36:12.027032145 +0000
***************
*** 59,66 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|0|M1|ROLHA|10|200|180|10|0
! VENDA|1|M1|VIDRO|25|25000|22500|5|0
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
--- 59,66 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|0|M1|ROLHA|10|200|200|10|0
! VENDA|1|M1|VIDRO|25|25000|25000|5|0
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
