*** tests/A-14-03-M-ok.out	2021-11-07 16:04:59.256980000 +0000
--- tests/A-14-03-M-ok.outhyp	2021-11-09 23:29:04.387368845 +0000
***************
*** 45,54 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
--- 45,55 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Abrir
  2 - Guardar
