*** tests/A-15-04-M-ok.out	2021-11-18 17:29:53.848835944 +0000
--- tests/A-15-04-M-ok.outhyp	2021-11-18 20:36:13.617030594 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|M1|GARRAFA|5|90|90|0|ROLHA:5:10#SAL:25:400#VIDRO:50:500
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: DESAGREGAÇÃO|0|M1|GARRAFA|5|970|970|0|ROLHA:5:10#SAL:25:10#VIDRO:50:10
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 50,61 ****
  0 - Sair
  Escolha uma opção: GARRAFA|S1|200|495
  HIDROGENIO|S1|200|5000
! ROLHA|M1|2|5
  ROLHA|R1|2|500
! SAL|M1|16|25
  SAL|S1|16|500
- VIDRO|M1|10|50
  VIDRO|M1|10|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 50,61 ----
  0 - Sair
  Escolha uma opção: GARRAFA|S1|200|495
  HIDROGENIO|S1|200|5000
! ROLHA|M1|2|1
  ROLHA|R1|2|500
! SAL|M1|18|5
  SAL|S1|16|500
  VIDRO|M1|10|500
+ VIDRO|M1|28|10
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
***************
*** 81,87 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|900|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 81,87 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
