*** tests/A-15-06-M-ok.out	2021-11-07 16:04:59.274650000 +0000
--- tests/A-15-06-M-ok.outhyp	2021-11-11 10:55:12.836946007 +0000
***************
*** 16,22 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Quantidade: Registar desagregação: Operação inválida: Produto 'GARRAFA': pedido=6, existências=5
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 16,31 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Gestão de Transacções
! 1 - Visualizar
! 2 - Registar desagregação
! 3 - Registar venda
! 4 - Registar compra
! 5 - Receber pagamento de venda
! 0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Opção inválida!
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
