*** tests/A-18-04-M-ok.out	2021-11-18 17:29:53.849835946 +0000
--- tests/A-18-04-M-ok.outhyp	2021-11-18 20:36:18.716025619 +0000
***************
*** 58,64 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|S1|GARRAFA|5|-20|0|0|ROLHA:20:40#VIDRO:10:30
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 58,64 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|S1|GARRAFA|5|-30|0|0|ROLHA:20:40#VIDRO:10:40
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
