*** tests/A-18-03-M-ok.out	2021-11-10 23:23:51.363056000 +0000
--- tests/A-18-03-M-ok.outhyp	2021-11-12 01:48:44.826445181 +0000
***************
*** 50,56 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|M2|GARRAFA|5|30|30|0|ROLHA:5:10#VIDRO:10:10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 50,56 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: DESAGREGAÇÃO|0|M2|GARRAFA|5|-150|10|0|ROLHA:1:40.0#VIDRO:2:40.0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
