*** tests/A-20-19-M-ok.out	2021-11-18 17:29:53.851835949 +0000
--- tests/A-20-19-M-ok.outhyp	2021-11-19 18:31:19.743218756 +0000
***************
*** 57,63 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|26000|0|2600|2600
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 57,63 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|26000|0|2600|2600
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
***************
*** 140,146 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M1|SAL|10|1000|1000|1|10
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 140,146 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M1|SAL|10|1000|1100|1|10
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 167,173 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|26000|0|3600|3600
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 167,173 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|36000|0|3600|3700
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
