*** tests/A-20-15-M-ok.out	2021-11-18 17:29:53.850835948 +0000
--- tests/A-20-15-M-ok.outhyp	2021-11-19 18:31:18.252220932 +0000
***************
*** 34,40 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|SAL|22|2200|2090|3
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 34,40 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|SAL|22|2200|2200|3
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 68,75 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|23000|0|2410|2300
! M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 68,75 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|24100|0|2410|2410
! M2|Toshiba|Tokyo, Japan|NORMAL|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
***************
*** 146,153 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|26000|0|2710|2600
! M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 146,153 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|27100|0|2710|2710
! M2|Toshiba|Tokyo, Japan|NORMAL|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
