*** tests/A-20-13-M-ok.out	2021-11-18 17:29:53.850835948 +0000
--- tests/A-20-13-M-ok.outhyp	2021-11-19 18:31:17.459222088 +0000
***************
*** 80,87 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|SELECTION|4100|0|410|410
! M2|Toshiba|Tokyo, Japan|SELECTION|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 80,87 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|4100|0|410|410
! M2|Toshiba|Tokyo, Japan|NORMAL|2100|0|210|210
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
