*** tests/A-20-02-M-ok.out	2021-11-18 17:29:53.850835948 +0000
--- tests/A-20-02-M-ok.outhyp	2021-11-19 18:31:14.118226961 +0000
***************
*** 97,103 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|1000|0|100|100
! M2|Toshiba|Tokyo, Japan|NORMAL|900|0|100|90
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 97,103 ----
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|1000|0|100|100
! M2|Toshiba|Tokyo, Japan|NORMAL|1000|0|100|90
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
