*** tests/A-20-11-M-ok.out	2021-11-18 17:29:53.850835948 +0000
--- tests/A-20-11-M-ok.outhyp	2021-11-19 18:31:16.648223271 +0000
***************
*** 178,184 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|SAL|1|100|160|2|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 178,184 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|2|M1|SAL|1|100|110|2|8
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 186,192 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|3|M2|SAL|1|100|115|8|11
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 186,192 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|3|M2|SAL|1|100|105|8|11
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 213,220 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|200|260
! M2|Toshiba|Tokyo, Japan|NORMAL|0|0|200|215
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 213,220 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|2000|0|200|210
! M2|Toshiba|Tokyo, Japan|NORMAL|2000|0|200|205
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
