*** tests/A-20-08-M-ok.out	2021-11-18 17:29:53.850835948 +0000
--- tests/A-20-08-M-ok.outhyp	2021-11-19 18:31:15.712224637 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|28|2800|2520|6
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|28|2800|2800|6
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 127,134 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|ELITE|25200|0|2800|2520
! M2|Toshiba|Tokyo, Japan|ELITE|28000|0|2800|2800
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
--- 127,134 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|28000|0|2800|2520
! M2|Toshiba|Tokyo, Japan|NORMAL|28000|0|2800|2800
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
